translations/messages.fr.yml

```diff
@@ -3,6 +3,8 @@
 #
 common.gender.man: Homme
 common.gender.woman: Femme
+common.gender.male: Homme
+common.gender.female: Femme
```

Comment:
where are these keys used?

Reply:
I add this to translate the Genders class

Reply:
Ok I see, there is `const CHOICES`, thank you